src/main/java/org/gitlab4j/api/RepositoryApi.java

```diff
@@ -1,5 +1,13 @@
 package org.gitlab4j.api;
 
+import org.gitlab4j.api.GitLabApi.ApiVersion;
+import org.gitlab4j.api.models.*;
```

Comment:
It is good practice to not use wildcard imports, please change this to only import the types being used.